src/nwbinspector/_configuration.py

```diff
@@ -0,0 +1,151 @@
+"""Primary functions for inspecting NWBFiles."""
+
+import json
+import jsonschema
+from pathlib import Path
+from enum import Enum
+from typing import Optional, List
+from types import FunctionType
+from packaging.version import Version
+
+import yaml
+
+from . import available_checks
+from ._registration import InspectorMessage, Importance
+from nwbinspector.utils._utils import (
+    PathType,
+)
+
+INTERNAL_CONFIGS = dict(dandi=Path(__file__).parent / "internal_configs" / "dandi.inspector_config.yaml")
+
+
+class InspectorOutputJSONEncoder(json.JSONEncoder):
```

Comment:
I suggest moving this to the `_formatting.py` file instead since it relates to the output.

Reply:
Done